tools/system_libs.py

```diff
@@ -1830,6 +1830,7 @@ def get_default_variation(cls, **kwargs):
       is_webgl2=settings.MAX_WEBGL_VERSION >= 2,
       is_ofb=settings.OFFSCREEN_FRAMEBUFFER,
       is_full_es3=settings.FULL_ES3,
+      is_enable_get_proc_address=settings.GL_ENABLE_GET_PROC_ADDRESS,
```

Comment:
This doubles the number of libgl variants from 16 to 32.. which is a little unfortunate but maybe not avoidable?

Reply:
The variants system we have is really good at managing the combinatorial complexity, so this does not double our maintenance work. Looking at disk, the getProcAddress function is the bulk of the code size, and the non-getProcAddress enabled variants of the files are mostly just 2KB in size (with exception of that one offscreenframebuffer file), so I don't think the disk size increase amounts to anything.